### ISSUE TITLE
The Event Sampler does not work with Alpha configuration IRFs 

### ISSUE THREAD

**Issue by fabiopintore** [gammapy/gammapy#3556]
**Gammapy version**
Present dev, v0.18.2.

**Bug description**
When using `MapDatasetEventSampler` with the new CTA Alpha configuration IRFs, an error is raised. 
In particular, the event sampler looks for specific keywords stored in `observation.aeff.`, which are however empty as they cannot be found in the mentioned IRFs. 
The related code is in `MapDatasetEventSampler.event_list_meta`:

```   
meta["ALTITUDE"] = observation.aeff.meta["CBD50001"][7:-4]
meta["ALT_PNT"] = observation.aeff.meta["CBD50001"][7:-4]
meta["AZ_PNT"] = observation.aeff.meta["CBD60001"][8:-4]
 ```

**To Reproduce**
Just follow the tutorial [https://docs.gammapy.org/0.18.2/tutorials/event_sampling.html](url) using the Alpha configuration IRFs.

Should we add a condition in the `event_list_meta` function to avoid missing keywords?

**Comment by adonath** [comment]
@fabiopintore I set the milestone for this to the next release. It's important that we fix this issue before the next CTA collaboration meeting.

**Comment by fabiopintore** [comment]
Thank you @adonath . I can fix the issue before the collaboration meeting. 
I've opened a PR (#) for including the new IRFs in `$GAMMAPY_DATA`. Any suggestion where adding the unit test?

However, I'm not convinced that we should only adapt the tutorial, as it would be just a patch for a specific case. Maybe we should consider to modify the `MapDatasetEventSampler`, following @AtreyeeS suggestion.

**Comment by adonath** [comment]
Fixed by #3568.
